File: InsuranceCompany.Core/Models/InsuranceStatus.cs
using System;
using System.Collections.Generic;

namespace InsuranceCompany.Core;

public partial class InsuranceStatus
{
    public Guid Id { get; set; }

    public string? Status { get; set; }

    public string? Color { get; set; }
    public bool? IsDisabledForms { get; set; }
    public bool? IsDisabledDocuments{ get; set; }

    public virtual ICollection<InsuranceRequest> InsuranceRequests { get; } = new List<InsuranceRequest>();
}

File: InsuranceCompany.Client/src/app/models/InsuranceStatus.ts
export class InsuranceStatus {
    id : string;
    status : string;
    color : string;
    isDisabledForms: boolean;

    constructor(id: string = '', status: string = '', color: string = '') {
        this.id = id;
        this.status = status;
        this.color = color;
        this.isDisabledForms = false;
    }
}